src/tests/cli.rs

```diff
@@ -25,7 +25,10 @@ fn bash_error_exit_code() -> Result<(), Box<dyn std::error::Error>> {
 
     // Changes locale to default to prevent locale-specific error messages.
     cmd.env("LC_ALL", "C")
-        .arg("--disable-format")
+        .arg("--disable-postprocessor")
+        .arg("shfmt")
+        .arg("--disable-postprocessor")
+        .arg("bshchk")
```

Comment:
We can do this, right? This way if we add more postprocessors to the list then we don't have to include them here.

```suggestion
        .arg("--disable-postprocessor")
        .arg("*")
```

Reply:
well in this way it is documented in the CLI without is not clear so much